src/evm/process-sponsor-wallet.ts

```diff
@@ -19,6 +19,7 @@ export const processSponsorWallet = async (
 
   // Keep track of nonce outside of the loop in case there is an invalid subscription and its nonce is skipped
   let nextNonce = transactionCount;
+
```

Comment:
I don't know if this comment is relevant in this PR but it seems that we've missed to cover lines 37-38 in the tests after last refactor.

Reply:
Do you mean testing a skipped invalid subscrptions? If so, I though the changes to [this test](https://github.com/api3dao/airkeeper/pull/50/files#diff-360f756b43e60a608f85b831e840b2ea931db0271eed2c3dc2ba021e45433942R202-R239) would be enough. Although I now realize I didn't update the name of the test to match.

Reply:
I mean when condition check doesn't result in a failure but instead returns false.